backend/install-sdk.sh

```diff
@@ -3,9 +3,9 @@
 set -e
 shopt -s expand_aliases
 
-if [ "$0" != "./build-sdk.sh" ]; then
+if [ "$0" != "./install-sdk.sh" ]; then
 	>&2 echo "Must be run from backend directory"
 	exit 1
 fi
 
-cargo install --bin=embassy-sdk --path=. --no-default-features --verbose
```

Comment:
No more verbose?

Reply:
i never saw the purpose of it. just a bunch of rustc output